main.go

```diff
@@ -857,6 +881,33 @@ func language(device ios.DeviceEntry, locale string, language string) {
 	fmt.Println(convertToJSONString(lang))
 }
 
+func assistiveTouch(device ios.DeviceEntry, operation string) {
+	wasEnabled, err := ios.GetAssistiveTouch(device)
+	var enable bool
+	exitIfError("failed getting current AssistiveTouch status", err)
+
+	if operation == "enable" {
```

Comment:
this might be nicer as a switch case like f.ex.
```
func main() {
    day := "Tue"
  
    // Use switch on the day variable.
    switch {
    case day == "Mon":
        fmt.Println("Monday")
    case day == "Tue":
        fmt.Println("Tuesday")
    case day == "Wed":
        fmt.Println("Wednesday")
    }
}
```